plugins/extensions/plugins/ZeroEQPlugin.cpp

```diff
@@ -334,6 +367,30 @@ bool ZeroEQPlugin::_requestFrameBuffers()
     return true;
 }
 
+bool ZeroEQPlugin::_requestSpikes()
+{
+    BRAYNS_INFO << "Spikes requested" << std::endl;
```

Comment:
Not really for now. We need a data provider service that does not yet exist. Experimental for now.